funnel_cake_index.gemspec

```diff
@@ -37,10 +37,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "traject", "~> 3.1"
+  spec.add_dependency "library_stdnums", "~> 1.6"
+  spec.add_runtime_dependency("gli", "~> 2.18")
+
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "traject", "~> 3.1"
   spec.add_development_dependency "rsolr", "~> 2.2"
```

Comment:
When I tried to build locally as a standalone gem, via https://stackoverflow.com/a/28758549, it fails to find rsolr dependency, so I think that needs to be a dependency as well.

Reply:
I migrated over instructions from `cob_index` to build `funnel_cake_index` locally for development. See if that works.